src/manage_sql.c

```diff
@@ -17810,9 +17810,9 @@ task_report_previous (task_t task, report_t report, report_t *previous)
                      "SELECT id FROM reports"
                      " WHERE task = %llu"
                      " AND scan_run_status = %u"
-                     " AND date < (SELECT date FROM reports"
+                     " AND creation_time < (SELECT creation_time FROM reports"
                      "             WHERE id = %llu)"
```

Comment:
Could you adjust the indentation here?

Reply:
the WHERE belongs to the subquery, so you should increase the indentation instead of removing it.